conformance/lang/expressions/list-constructor-expr/spread_operator.balt

```diff
@@ -0,0 +1,728 @@
+Test-Case: output
+Description: Test spread operator with basic contextually expected type.
+Labels: array-type, list-constructor-expr, spread-operator, tuple-type
+
+function init() {
+    int[] a0 = [];
+    int[2] a1 = [];
+    int[2] a2 = [1, 2];
+    int[] v0 = [...a0];
+    [int, int] v1 = [...a1];
+    int[] v2 = [...a2];
+    io:println(v0); // @output []
+    io:println(v1); // @output [0,0]
+    io:println(v2); // @output [1,2]
+    io:println(v2); // @output [1,2]
```

Comment:
Better if we can print the value in the immediate line. Easy to read and we have followed that pattern in the other spec conformance tests as well IINM.
```suggestion
    int[] a0 = [];
    int[2] a1 = [];
    int[2] a2 = [1, 2];
    
    int[] v0 = [...a0];
    io:println(v0); // @output []
     
    [int, int] v1 = [...a1];
    io:println(v1); // @output [0,0]
    
    int[] v2 = [...a2];
    io:println(v2); // @output [1,2]
```